exercises/crypto-square/canonical-data.json

```
@@ -10,6 +10,15 @@
       },
       "expected": ""
     },
+    {
+      "uuid": "aad04a25-b8bb-4304-888b-581bea8e0040",
+      "description": "empty plaintext after normalized",
```

I would suggest the following tweak to the description, to be more consistent with other descriptions in the file:

```suggestion
      "description": "normalization results in empty plaintext",
```


Note that the descriptions are not all consistent in this file, but most of them are some variation on "x results in y":

```
$ cat exercises/crypto-square/canonical-data.json | jq .cases[].description
"empty plaintext results in an empty ciphertext"
"Lowercase"
"Remove spaces"
"Remove punctuation"
"9 character plaintext results in 3 chunks of 3 characters"
"8 character plaintext results in 3 chunks, the last one with a trailing space"
"54 character plaintext results in 7 chunks, the last two with trailing spaces"
```

---

I pushed the "Commit suggestion" button, not sure if I was suppose to do that... But of course your suggestion sound good to me.